pioneers-testnet/pioneers-exercise-2.md

```diff
@@ -70,12 +74,16 @@ In this excercise we will be following steps from [Creating a Simple Transaction
    Create a new address *myaddr2*:
 
         cardano-cli shelley address key-gen …
-        cardano-cli shelley address build-enterprise …
+        cardano-cli shelley address build …
 
 3. Build a transaction to transfer Ada from *myaddr* to *myaddr2*.  You will need to use the Shelley transaction processing operations:
```

Comment:
Should we add a step to fetch protocol params, calculate min fee and get current tip instead of picking overly large numbers?

Reply:
might be better yes, as in the tx.md tutorial we are already checking for fee.
Thinking of it, maybe we need to check the TTL in both this one and in tx.md?

Reply:
For TTL: Query tip or query prometheus metrics ?